### ISSUE TITLE
Make edx-ci sandbox for Teacher's Portal

### ISSUE THREAD

**Issue by pdpinch** [mitocw/edx-platform#178]

**Comment by pdpinch** [comment]
blocked by #179

**Comment by pdpinch** [comment]
enable ccx
import course
connect to teachersportal-ci

**Comment by blarghmatey** [comment]
Do we still need this @pdpinch @bdero?

**Comment by bdero** [comment]
My thought is that this will probably be needed the next time teachersportal receives some more stories, but for the time being it seems low priority.
